README.md

```diff
@@ -61,7 +61,7 @@ You can change the HTML/CSS layout if you need it.
 ## Deploy and Pull Request
 
 1. Replace `<your_account>` with your Github username in the link
-    - [DEMO LINK](https://<your_account>.github.io/js_2048_game/)
+    - [DEMO LINK](https://mmromanova.github.io/js_2048_game/)
```

Comment:
Add this demo link to the PR description 
<img width="943" alt="image" src="https://github.com/user-attachments/assets/6a0368a5-5cfb-4826-b5fc-96bc9ce0ae0a" />